src/main/java/com/graphhopper/converter/api/GisgraphySearchEntry.java

```diff
@@ -35,6 +35,9 @@
     @JsonProperty("adm1_name")
     private String adm1Name;
 
+    @JsonProperty("adm3_name")
```

Comment:
I wonder that the `@JsonProperty` also for the other properties in this class work. For example see [this](https://services.gisgraphy.com/reversegeocoding/?format=json&apikey=graphHopperTestKey&lat=48.4882&lng=2.6996) request. The fields in json are named: `adm3Name` or `zipCode`. But as I mentioned, I wasn't able to test Gisgraphy (see #48).